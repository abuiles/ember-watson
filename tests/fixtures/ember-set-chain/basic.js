function fn() {
  x.set('foo', 0);
  x.set('foo', 0).set('bar', 1).set('baz', 2);
  var y = x.set('foo', 0);
  for (;x.set('foo', 0);) {};
  return x.set('foo', 0);
}
