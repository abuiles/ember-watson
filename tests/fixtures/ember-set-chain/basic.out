Using the return value of `set`:

 2 |   x.set('foo', 0);
 3 |   x.set('foo', 0).set('bar', 1).set('baz', 2);
   |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 4 |   var y = x.set('foo', 0);

Using the return value of `set`:

 2 |   x.set('foo', 0);
 3 |   x.set('foo', 0).set('bar', 1).set('baz', 2);
   |   ^^^^^^^^^^^^^^^
 4 |   var y = x.set('foo', 0);

Using the return value of `set`:

 3 |   x.set('foo', 0).set('bar', 1).set('baz', 2);
 4 |   var y = x.set('foo', 0);
   |           ^^^^^^^^^^^^^^^
 5 |   for (;x.set('foo', 0);) {};

Using the return value of `set`:

 4 |   var y = x.set('foo', 0);
 5 |   for (;x.set('foo', 0);) {};
   |         ^^^^^^^^^^^^^^^
 6 |   return x.set('foo', 0);

Using the return value of `set`:

 5 |   for (;x.set('foo', 0);) {};
 6 |   return x.set('foo', 0);
   |          ^^^^^^^^^^^^^^^
 7 | }
